test/oli_web/live/remix_section_test.exs

```diff
@@ -145,6 +145,49 @@ defmodule OliWeb.RemixSectionLiveTest do
     end
   end
 
+  describe "breadcrumbs" do
+    setup [:setup_session]
+
+    test "as instructor", %{
+      conn: conn,
+      map: %{
+        section_1: section_1
+      }
+    } do
+      conn =
+        get(conn, Routes.live_path(OliWeb.Endpoint, OliWeb.Delivery.RemixSection, section_1.slug))
+
+      {:ok, _view, html} = live(conn)
+
+      refute html =~
```

Comment:
Anyone reason to not put these on one line? Feels like it is adding lines for no reason.

Reply:
It's the auto format on save that leaves it like that. Maybe I can see if is a specific rule I can change